File: Data Structures & Algorithms/find-all-numbers-disappeared-in-an-array/submission-0.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        int n=nums.size();
        vector<int>result;
        for(int i=0;i<n;i++){
            int temp=abs(nums[i])-1;
            if(nums[temp]>0)
            nums[temp]=-nums[temp];
        }
        for(int i=0;i<n;i++){
            if(nums[i]>0) result.push_back(i+1);
        }
        return result;
    }
};

File: Data Structures & Algorithms/special-array-i/submission-0.py
class Solution:
    def isArraySpecial(self, nums: List[int]) -> bool:
        
        new_lst = list(map(lambda x: (x%2 == 0),nums))
        for i in range(len(nums)-1):
            if new_lst[i] == new_lst[i+1]:
                return False

        return True

